docusaurus.config.js

```diff
@@ -39,6 +39,7 @@
         }),
       ],
     ],
+    themes: ['@docusaurus/theme-search-algolia'],
```

Comment:
remove it

Reply:
remove line 42, the theme is already in the preset-classic

Reply:
done